apps/docs-snippets/src/guide/transactions/transaction-request.test.ts

```diff
@@ -173,7 +173,7 @@ describe('Transaction Request', () => {
     });
 
     const predicateCoins = await predicate.getResourcesToSpend([
-      { amount: 1000, assetId: baseAssetId },
+      { amount: 2000, assetId: baseAssetId },
```

Comment:
This is just to reduce probability of a failure?

Reply:
@maschad Yes